proxy/proxy_test.go

```diff
@@ -133,6 +135,34 @@ func (srw *syncResponseWriter) Len() int {
 	return srw.body.Len()
 }
 
+func containsCount(s, substr string, count int) bool {
+	var found int
+	for {
+		i := strings.Index(s, substr)
+		if i < 0 {
+			return found == count
+		}
+
+		if found == count {
+			return false
+		}
+
+		found++
+		s = s[i+len(substr):]
+	}
+}
+
+func containsEveryLineCount(s, substr string, count int) bool {
+	l := strings.Split(substr, "\n")
+	for _, li := range l {
+		if !containsCount(s, li, count) {
```

Comment:
Are you sure we want count per line and not just a total count in s?

Reply:
Dropped this function.